recipe/meta.yaml

```diff
@@ -23,7 +23,6 @@ build:
   script:
     # The issue of some CUDA headers being moved out of /usr/local/cuda/ since 10.1+
     # is addressed in conda-forge/docker-images#134, so we do nothing in the recipe
-    
```

Comment:
Would it make sense to just drop this? Also CUDA 11 does not have this issue either.

Reply:
Sounds good.